app/assets/stylesheets/provider/_buttons.scss

```diff
@@ -84,37 +73,34 @@ form.formtastic .button-bar, .button-bar {
 
   & > fieldset {
 
-    &.actions:first-child {
-      position: absolute;
-      left: 0;
+    &.actions:last-child {
+      float : right;
       margin: line-height-times(1/2) 0 0 0;
       padding: 0 0 0 0;
-      text-align: left !important;
 
       & > ol > li {
-        margin-right: line-height-times(1/3);
+        margin-left: line-height-times(1/3);
       }
+
     }
 
-    &.actions:last-child {
-      position: absolute;
-      width: 65%;
-      right: 0;
+    &.actions:first-child {
+      float: left;
       margin: line-height-times(1/2) 0 0 0;
       padding: 0 0 0 0;
+      text-align: left !important;
 
       & > ol > li {
-        margin-left: line-height-times(1/3);
+        margin-right: line-height-times(1/3);
       }
-
     }
 
     & > ol > li { display: inline-block; }
   }
 }
 
-.formtastic.cms_template .button-bar {
-  height: line-height-times(1.5);
+.formtastic .button-bar {
+  display: flow-root;
```

Comment:
This permits us using `float: left` and `float: right` for "actions" divs instead of `position: absolute`, by creating a new block formatting context.